packages/e2e-tests/register/seedPhrase.spec.js

```diff
@@ -0,0 +1,125 @@
+const { test, expect } = require("@playwright/test");
+
+const {
+    generateTestAccountId,
+    connectToAccountWithSeedphrase,
+} = require("../utils/account");
+const { HomePage } = require("./models/Home");
+const { CreateAccountPage } = require("./models/CreateAccount");
+const { SetRecoveryOptionPage } = require("./models/SetRecoveryOption");
+const { SetupSeedPhrasePage } = require("./models/SetupSeedPhrase");
+const { VerifySeedPhrasePage } = require("./models/VerifySeedPhrase");
+
+const { describe, afterAll } = test;
+
+describe("Account Registration Using Seed Phrase", () => {
+    const testAccountId = generateTestAccountId();
+    let testAccountInstance;
+
+    afterAll(async () => {
+        testAccountInstance &&
+            (await testAccountInstance.deleteAccount("testnet"));
```

Comment:
The account that we sweep funds to should probably be the `BANK_ACCOUNT` (that way we send funds back to the bank account if we are running on mainnet)